axelrod/match.py

```diff
@@ -119,6 +119,6 @@ def play(self):
 
     def sparklines(self, c_symbol=u'█', d_symbol=u' '):
         return (
-            sparkline(self._player1.history, c_symbol, d_symbol) +
+            sparkline(self.player1.history, c_symbol, d_symbol) +
             u'\n' +
-            sparkline(self._player2.history, c_symbol, d_symbol))
+            sparkline(self.player2.history, c_symbol, d_symbol))
```

Comment:
Even though we don't necessarily want it exposed to the user? (Or perhaps we do?)

Genuine questions, not at all disagreeing, just wanting to learn :)